samples/snippets/long_running_operation.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+""" DialogFlow CX long running operation code snippet """
+
+## [START dialogflow_cx_long_running_snippet]
+from google.cloud.dialogflowcx_v3.services.agents.client import AgentsClient
+from google.cloud.dialogflowcx_v3.types.agent import ExportAgentRequest
+
+
+def export_agent(project_id, agent_id, location):
```

Comment:
Could you elaborate the method name to reflect it being part of LRO function? Something like `export_long_running_agent` I think would help make it more descriptive and fitting in this file.

Mentioning this because I'm having difficulty finding correlation between `long_running_operation` and `export_agent`. It could just be me not having familiarity in this concept however.